scripts/ubuntu/ganeti.sh

```diff
@@ -1,3 +1,3 @@
-apt-get -y install pump cloud-utils cloud-init cloud-initramfs-growroot \
+apt-get -y install cloud-utils cloud-init cloud-initramfs-growroot \
```

Comment:
Let's do Ubuntu in a separate PR in case there are updates we need to do on those releases.